### PR TITLE
OPENEUROPA-1992: ECL 2.0: Deprecate site switcher

### PR DESCRIPTION
**Pull request by voidtek**
### Description

Remove the block and the template for Corporate Blocks Site Switcher.

### CODE REVIEW

File: modules/oe_theme_helper/oe_theme_helper.post_update.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ function oe_theme_helper_post_update_use_retina_image_styles(array &$sandbox): v
 /**
  * Change the region of the search block.
  */
-function oe_theme_helper_post_update_8201_change_region_of_search_block() {
+function oe_theme_helper_post_update_00001_change_region_of_search_block() {
```
Comment (brummbar) [inline modules/oe_theme_helper/oe_theme_helper.post_update.php]: Let's update this to the new format with only the 00001.

Comment (brummbar) [reply modules/oe_theme_helper/oe_theme_helper.post_update.php]: Same for the next 00002.

File: README.md [hunk 2]
```diff
@@ -300,6 +300,10 @@ Note that the following models are deprecated after upgrading from 1.x to 2.x:
 - `dialog pattern`
 - `file_link pattern`
 
+Note that the following blocks are deprecated after upgrading from 1.x to 2.x:
```
[the remark this answers is not in the document]
Comment (voidtek) [reply README.md]: done.

File: modules/oe_theme_helper/oe_theme_helper.post_update.php (hunk 1 above)
Comment (voidtek) [reply modules/oe_theme_helper/oe_theme_helper.post_update.php]: done.

**Pull request merged**